moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -204,7 +204,11 @@ void ChompOptimizer::initialize()
   for (size_t i = 0; i < joint_model_group_->getFixedJointModels().size(); i++)
   {
     const moveit::core::JointModel* model = joint_model_group_->getFixedJointModels()[i];
```

Comment:
Please take the chance and replace the for-loop by `for (const moveit::core::JointModel* model : joint_model_group_->getFixedJointModels())`.